pkgs/development/compilers/cudatoolkit/default.nix

```diff
@@ -56,5 +57,19 @@ in rec {
     gcc = gcc9;
   };
 
+  cudatoolkit_11_3 = common {
+    version = "11.3.1";
+    url = "https://developer.download.nvidia.com/compute/cuda/11.3.1/local_installers/cuda_11.3.1_465.19.01_linux.run";
+    sha256 = "sha256:0d19pwcqin76scbw1s5kgj8n0z1p4v1hyfldqmamilyfxycfm4xd";
```

Comment:
Please remove the sha256: prefix.

Reply:
Done in https://github.com/NixOS/nixpkgs/pull/136655/commits/09b33fe7b4412dad09a7aecd95bbe54364b71567.